media-libs/amf-headers/Manifest

```diff
@@ -1,2 +1 @@
-DIST amf-headers-1.4.33.tar.gz 198142272 BLAKE2B 65e616012463e21bfa113ff62f76baaf186e18c6b3422bf5b2fdf8d208efe0ea7d07b74d6d292bc391202d2190c871dd0d6f997783b3ffa533d4086e395ae67f SHA512 e0c4f583996ff2d1d61c53b8ce7ef2eadb32d2a13930e59146b935840e31d032c5cec48baced70b0007fa5f33e30537d03ddf71140ff51213085aba20e16f5ca
```

Comment:
This is the last stable version of amf-headers, I have to leave this version to the tree until 1.4.34 is stabilized.

Reply:
Thanks. Will you stabilize that old package version ?